include/megaapi.h

```diff
@@ -5479,6 +5479,8 @@ class MegaApi
          */
         static void setLogLevel(int logLevel);
 
+        static void setLogToConsole(bool enable);
```

Comment:
missing docstrings saying that this determines if log is printed in console in the absence of a logger object. By default true. And it is likely to change in the future.